recipes/pointpats/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "pointpats" %}
+{% set version = "2.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3aaceb5576b8a587e5b8b81877dfd0b87439a199e8a813af27086effbd9ed673
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - python >=3.4
+    - scipy>=0.11
```

The version constraints need to be formatted like this:
```suggestion
    - scipy >=0.11
```
i.e. one space after the name and nothing after the operator (`>`)

---

awesome, thanks!